dev/ProgressBar/ProgressBar.xaml

```diff
@@ -105,7 +105,7 @@
                                 <VisualState x:Name="Updating" />
                                 <VisualState x:Name="UpdatingError">
                                     <VisualState.Setters>
-                                        <Setter Target="DeterminateProgressBarIndicator.Fill" Value="{ThemeResource ProgressBarErrorForegroundColor}" />
+                                        <Setter Target="DeterminateProgressBarIndicator.(Shape.Fill).(SolidColorBrush.Color)" Value="{ThemeResource ProgressBarErrorForegroundColor}" />
```

Comment:
While this works, will this crash if someone uses lightweight styling to override the ProgressBarForeground resource with a Brush that's not a SolidColorBrush?

Should this just be

<Setter Target="DeterminateProgressBarIndicator.Fill" Value="{ThemeResource ProgressBarErrorForeground}" />

assuming that Brush exists?

Reply:
@bschoepke this brush doesn't exist : ) we only have the Color resource for the Error and Paused states:
![image](https://user-images.githubusercontent.com/21356912/124670169-6f49ba80-de68-11eb-977c-2d7777a4aabe.png)

Unfortunately, we cannot simply use **.Fill** property because in all other animations in progressBar  **.Color** property is used to animate the color and setting just the **.Fill** in the visual state won't ever get unset to the default value and we'll be stuck with it :)

Reply:
I think we might have moved from brush to color to get brush transitions to work. This is unrelated to this change but we can follow up on this.

Reply:
Correct, color animations only work with colors.